src/components/Releases/index.tsx

```diff
@@ -14,9 +14,13 @@ type ReleaseData = {
     signature: string,
 }
 
-function createReleaseData(version: string): ReleaseData {
-    const vtag = `${version}-incubating`
-    const archive = `https://downloads.apache.org/incubator/kvrocks/${version}/apache-kvrocks-${vtag}-src.tar.gz`
+function createReleaseData(index: number, version: string): ReleaseData {
+    var vtag = `${version}-incubating`
+    // Drop the incubating suffix for releases after graduation
+    if (index >= 4) {
+        vtag = `${version}`
+    }
```

Comment:
Why not inline the version to the `versions` list so that we don't do if-else here?